examples/advanced/network.tf

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
What was the motivation for creating this file?

Reply:
I think this question is not to me, because not I created this file
https://github.com/terraform-google-modules/terraform-google-cloud-nat/commit/56f14e693a9a2aa9cffd6b55676dee3802faac01

Reply:
@ingwarr this is due to you adding some random commit to the PR

Reply:
@ingwarr this file is still present on your branch. It is not present on [master](https://github.com/terraform-google-modules/terraform-google-cloud-nat/tree/master/examples/advanced) or on [aaron-lane-0.12](https://github.com/terraform-google-modules/terraform-google-cloud-nat/tree/aaron-lane-0.12/examples/advanced). Please remove it if it was added erroneously as @bohdanyurov-gl suggested.

Reply:
I've removed it